android/java/org/chromium/chrome/browser/crypto_wallet/adapters/WalletCoinAdapter.java

```diff
@@ -139,6 +122,29 @@ public void onBindViewHolder(@NonNull ViewHolder holder, int position) {
             }
         });
 
+        if (walletListItemType == Utils.TRANSACTION_ITEM) {
+            holder.txStatus.setVisibility(View.VISIBLE);
+            holder.txStatus.setText(walletListItemModel.getTxStatus());
+            holder.txStatus.setCompoundDrawablesRelativeWithIntrinsicBounds(
+                    new BitmapDrawable(
+                            context.getResources(), walletListItemModel.getTxStatusBitmap()),
+                    null, null, null);
+            holder.feeText.setVisibility(View.VISIBLE);
+            holder.feeText.setText(String.format(
+                    context.getResources().getString(R.string.wallet_tx_fee),
+                    String.format(Locale.getDefault(), "%.6f", walletListItemModel.getTotalGas()),
+                    walletListItemModel.getChainSymbol(),
+                    String.format(
+                            Locale.getDefault(), "%.2f", walletListItemModel.getTotalGasFiat())));
+            Utils.overlayBitmaps(mExecutor, mHandler, walletListItemModel.getAddressesForBitmap(),
+                    holder.iconImg);
+            if (walletListItemModel.getTransactionInfo().txStatus == TransactionStatus.REJECTED) {
+                holder.disableClick();
+            } else {
+                holder.itemView.setClickable(true);
+            }
```

Comment:
I think don't need the else part now

Reply:
If an item(REJECTED) is recycled for an item with status other than `REJECTED` when scrolled?

Reply:
https://github.com/brave/brave-core/pull/14279#discussion_r926880397